main.go

```diff
@@ -81,7 +84,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	kubeconfigHandler, err := config.NewKubeconfigBuilder(configprovider.ViperKey)
+	kubeconfigHandler, err := config.NewKubeconfigBuilder(configprovider.InMemKey)
```

Comment:
Helm installations doesn't quite works with in-memory stores. Please use meshkit's viper config provider (refer: https://github.com/meshery/meshery-kuma/pull/193)

Reply:
Done

Reply:
Thanks for restoring `ViperKey` however as this is still using `meshery-adapter-library`'s `config` handlers, we will have data race issues. Let's track this issue in another issue.

Reply:
Currently Viper key is not written/changed by any function. So is there still a chance for data race issue?

Reply:
You are right. This isn't where we will get data race issue. However, `ViperKey` implies that we are instructing adapter to use `viper` as its config provider. This seems and works fine but starts panicking (due to data race) when concurrent actions are performed on the adapter. Adapter, in order to perform these actions uses its config provider to R/W some data, by default `viper` is **not** thread safe, hence it panics. Meshkit's viper wrapper deals with this thread unsafe nature.